lib/minitest/test.rb

```diff
@@ -18,6 +18,12 @@ class << self; attr_accessor :io_lock; end # :nodoc:
     self.io_lock = Mutex.new
 
     ##
+    # The signal to use for dumping information to STDERR. Defaults to "INFO".
+
+    class << self; attr_accessor :info_signal; end
+    self.info_signal = "INFO"
+
```

Comment:
I wasn't sure where to put it and was hoping for feedback. Where would you prefer?